apps/opik-frontend/src/components/pages/TracesPage/AddToDataset/AddToDatasetDialog.tsx

```diff
@@ -82,8 +82,10 @@ const AddToDatasetDialog: React.FunctionComponent<AddToDatasetDialogProps> = ({
             const traceId = isSpan ? get(r, "trace_id", "") : r.id;
 
             return {
-              input: r.input,
-              expected_output: r.output,
+              data: {
+                input: r.input,
```

Comment:
Is input a required field in the trace, right?

Reply:
I don't know this information but in this case it doesn't matter, we filter out these traces or spans. 

```
const validRows = useMemo(() => {
    return rows.filter((r) => isObject(r.input));
  }, [rows]);
```